partiql-planner/src/main/kotlin/org/partiql/planner/internal/typer/PlanTyper.kt

```diff
@@ -817,7 +819,16 @@ internal class PlanTyper(
         }
 
         override fun visitRexOpPivot(node: Rex.Op.Pivot, ctx: StaticType?): Rex {
-            TODO("Type RexOpPivot")
```

Comment:
Since `PIVOT` is now implemented, perhaps need to mention in the CHANGELOG?

Reply:
This is grouped with adding PartiQLPlanner